vars/main.yml

```diff
@@ -23,6 +23,12 @@ archivematica_src_ss_gunicorn_config: "/etc/archivematica/storage-service.gunico
 archivematica_src_am_amauat_deps: []
 archivematica_src_am_fixity_deps: []
 
+archivematica_fixity_cron_minute: "0"
```

Comment:
The vars/main.yml is not the best place to put the variables when you want to allow the change when settings in host_vars or group_vars.


See: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#understanding-variable-precedence

I'd use defaults/main.yml instead to set the default values.